examples/panda3d-character-controller/main.py

```diff
@@ -60,6 +61,7 @@ def post_load_hook(self, node, entity):
     panda3d.ExecuteMovement,
     # We're done with character movement, now adjust the cameras.
     panda3d.UpdateCameras,
+    panda3d.CameraCollisions,
```

Comment:
For naming consistency, `CollideCameras`? Systems are verbs (or verbial phrases), not nouns.